src/screens/AccountDetails.js

```diff
@@ -71,7 +71,7 @@ class AccountDetailsView extends React.Component {
 This account can only be recovered with its associated recovery phrase.`,
       [
         {
-          text: 'Delete',
+          text: 'Delete & Archive Account',
```

Comment:
I'd for for Archive only since we're not deleting it (yet
```suggestion
          text: 'Archive',
```